canister_e2e_ids.json

```diff
@@ -2,6 +2,9 @@
 	"backend": {
 		"local": "tdxud-2yaaa-aaaad-aadiq-cai"
```

Comment:
should the backend be updated too? it changed?

Reply:
I can be wrong, but I think this is kind of a temporary solution - same as locally, to make backend work, we need to manually update the signer canister id value in `.dfx/local/canister_ids.json` with the same id as `backend` canister has. My assumption is that when the old backend is completely removed, new backend and singer will have separate ids.